x-pack/plugins/canvas/canvas_plugin_src/uis/views/shape.js

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-import { shapes } from '../../renderers/shape/shapes';
+import { shapes } from '../../../../../../src/plugins/expression_shape/public';
```

Comment:
Ok so it seems like this is the point that canvas needs to pull in all of the shapes and the reason for the shapes to be exported from the plugin.  

Can we instead of the expression_shape plugin export 
1) A list of available shapes
2) A component that will accept a shapeType prop, and maybe whatever dimensions we need to give it, and it will render the shape.  This component could lazy/suspense load the shapes.  

Then, I think, Canvas can get the list of shapes, and maybe changes it's ShapePreview to use that shape render suspense component.  Does that make sense?

Reply:
Yes, sure, I'd prefer such a workflow.

Reply:
In this way we can't do that, because every component is manipulating with `svg` data differently. But I'm thinking on the solution right now.

Reply:
I've provided the solution in some last commits.